app/src/app/components/login/BasicUserAuthForm.tsx

```diff
@@ -40,48 +43,54 @@ export const BasicUserAuthForm = () => {
       navigate("/");
     } catch (error) {
       console.error(error);
-      if (error instanceof ApiError) {
-        if (error.status === 401) {
-          form.setError("email", { message: "Invalid email or password" });
-          form.setError("password", { message: "Invalid email or password" });
+      if (error instanceof ApiError && error.status === HttpStatus.Unauthorized) {
+        if (error.message.includes("must change your password")) {
+          return navigate(`/update-password?email=${values.email}&error=${error.message}`);
```

Comment:
So a non-new user *could* navigate here and change their password that way.?

Reply:
yeah if they want